packages/ui/src/components/StoryCard/StoryCard.tsx

```diff
@@ -36,14 +37,16 @@ export function StoryCard({
         <div
             className={twMerge(
                 'flex flex-row items-center p-0 bg-white group sm:max-w-[530px]',
-                size === 'large' ? `flex-col gap-0` : `md:flex-row items-start gap-4`,
+                size === 'large' ? `flex-col gap-0` : `md:flex-row sm:max-w-none items-start gap-4`,
                 className,
             )}
         >
             <Link
                 className={twMerge(
                     'aspect-[10/7]',
-                    size === 'large' ? 'w-full md:aspect-[27/17]' : 'w-1/3 hidden md:block',
+                    size === 'large'
+                        ? 'w-full md:aspect-[27/17]'
+                        : 'w-1/3 hidden md:block md:max-w-[200px]',
```

Comment:
One approach that I have seen in many libraries/apps using `twMerge` is to introduce a `cn()` utility which leverages `clsx` or `classNames` within `twMerge`:

```tsx
import { clsx, type ClassValue } from "clsx";
import { twMerge } from "tailwind-merge";

export function cn(...inputs: ClassValue[]) {
  return twMerge(clsx(inputs));
}
```

Then you can still use the good old way of doing styles like we used to do before using the objects presentations.

```tsx
cn('aspect-[10/7]', { "w-full md:aspect-[27/17]": size === 'large' })
```

Bounce point, it's _shorter_ :)

(Somehow a big deal for a utility which gets reused a lot)

What do you think Francis?

Reply:
👍🏾 I do agree, and it keeps it consistent with how we've been doing it earlier, I would leave it for later though as we'll have to change most parts where we use this syntax